SignalRServiceBenchmarkPlugin/plugins/Plugin.Microsoft.Azure.SignalR.Benchmark/SignalRPlugin/SlaveMethods/CreateConnection.cs

```diff
@@ -54,25 +59,23 @@ private IList<HubConnection> CreateConnections(int total, string url, string tra
             success = Enum.TryParse<HttpTransportType>(transportTypeString, true, out var transportType);
             PluginUtils.HandleParseEnumResult(success, transportTypeString);
 
-            var connections = Enumerable.Repeat<HubConnection>(null, total);
-            connections =
-            from connection in connections
-            let cookies = new CookieContainer()
-            let httpClientHandler = new HttpClientHandler
-            {
-                ServerCertificateCustomValidationCallback = HttpClientHandler.DangerousAcceptAnyServerCertificateValidator,
-                CookieContainer = cookies,
-            }
-            let hubConnectionBuilder = new HubConnectionBuilder()
-                .WithUrl(url, httpConnectionOptions =>
-                {
-                    httpConnectionOptions.HttpMessageHandlerFactory = _ => httpClientHandler;
-                    httpConnectionOptions.Transports = transportType;
-                    httpConnectionOptions.CloseTimeout = TimeSpan.FromMinutes(100);
-                    httpConnectionOptions.Cookies = cookies;
-                })
-            select
-            protocolString.ToLower() == "messagepack" ? hubConnectionBuilder.AddMessagePackProtocol().Build() : hubConnectionBuilder.Build();
+            var connections = from i in Enumerable.Range(0, total)
+                              let cookies = new CookieContainer()
+                              let httpClientHandler = new HttpClientHandler
+                              {
+                                  ServerCertificateCustomValidationCallback = HttpClientHandler.DangerousAcceptAnyServerCertificateValidator,
+                                  CookieContainer = cookies,
+                              }
+                              let hubConnectionBuilder = new HubConnectionBuilder()
+                                  .WithUrl(url, httpConnectionOptions =>
+                                  {
+                                      httpConnectionOptions.HttpMessageHandlerFactory = _ => httpClientHandler;
+                                      httpConnectionOptions.Transports = transportType;
+                                      httpConnectionOptions.CloseTimeout = TimeSpan.FromMinutes(_closeTimeout);
+                                      httpConnectionOptions.Cookies = cookies;
+                                  })
+                              select
+                              protocolString.ToLower() == "messagepack" ? hubConnectionBuilder.AddMessagePackProtocol().Build() : hubConnectionBuilder.Build();
```

Comment:
```cs
var connections = from i in Enumerable.Range(0, total)
                  select new HttpClientHandler { ... } into handler
                  select new HubConnectionBuilder().WithUrl( ... ) into builder
                  select ...
```

Reply:
The `cookies` has to be used in two expression followed, thus it needs to store result of a sub-expression.